src/NzbDrone.Core/ImportLists/Simkl/SimklParser.cs

```diff
@@ -22,22 +29,48 @@ public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse import
                 return series;
             }
 
-            var jsonResponse = STJson.Deserialize<SimklResponse>(_importResponse.Content);
+            var jsonResponse = Json.Deserialize<SimklResponse>(_importResponse.Content);
 
             // no shows were return
             if (jsonResponse == null)
             {
                 return series;
             }
 
-            foreach (var show in jsonResponse.Shows)
+            if (jsonResponse.Anime != null)
+            {
+                foreach (var show in jsonResponse.Anime)
+                {
+                    var tentativeTvdbId = int.TryParse(show.Show.Ids.Tvdb, out var tvdbId) ? tvdbId : 0;
+
+                    if (tentativeTvdbId > 0 && show.AnimeType.Equals(SimklAnimeType.Tv.ToString().ToLowerInvariant()))
```

Comment:
Along with changing the prop to the correct enum:

```suggestion
                    if (tentativeTvdbId > 0 && show.AnimeType == SimklAnimeType.Tv)
```